other/react-native/__tests__/__snapshots__/render-tests.js.snap

```diff
@@ -9,9 +9,7 @@ exports[`renders with React Native components 1`] = `
     aria-expanded={false}
     autoComplete="off"
     id="downshift-0-input"
-    onBlur={[Function]}
```

Comment:
Hmmm... It feels like this would be a regression.

Reply:
Shouldn't this snapshot (the react native snapshot) reflect that the code in the react native build doesn't handle those props / that they won't be there? I could pass through noops but I don't think that'd be right.

Reply:
Right, but before your changes, the input had these event handlers for react-native and after your chances it doesn't. That's the regression I think.

Reply:
Agreed. We should ensure that the onBlur handler is still passed here.